### DISCUSSION TITLE
Viewing Sioyek highlights and notes on tablets?

### DISCUSSION THREAD

**Discussion by MadsJensen** [ahrm/sioyek#470] [General]
Hi all, 

Are there any plans for tablet version or is a way to view highlights and notes from sioyek on tablets?

**Comment by ahrm** [comment]
Currently there are no plans for a tablet version.
You can either use [`embed_annotations`](https://sioyek-documentation.readthedocs.io/en/latest/commands.html#embed-annotations) command to export a new version of PDF files with embedded annotations or use the [`embed_annotations` extension](https://github.com/ahrm/sioyek-python-extensions#-embed_annotations) to embed the annotations in the current document.

**Comment by MadsJensen** [comment]
thank you very much for the very quick reply! 

That makes sense and those are good options. Thanks for pointing them out -- I missed them before.
